EMaligner/EMaligner.py

```diff
@@ -313,8 +327,13 @@ def assemble_from_hdf5(self, filename, zvals, read_data=True):
             if "results" in f.keys():
                 results = json.loads(f.get('results')[()][0].decode('utf-8'))
 
-            r = json.loads(f.get('resolved_tiles')[()][0].decode('utf-8'))
-            self.resolvedtiles = renderapi.resolvedtiles.ResolvedTiles(json=r)
+            r = f.get('resolved_tiles')[()][0]
```

Comment:
does this break support for previously-generated h5 files?

Reply:
Yes, though, I would probably be the only user who was exploring that, and I've been working off the dev branches for distributed for some time, partly for the reason that these objects did not serialize/deserialize generally when embedded into the h5 file.